backend/editor/entries.py

```diff
@@ -242,3 +242,38 @@ def update_node_children(entry, new_children_ids):
         result = session.run(query, {"id": entry, "child": child})
     
     return result
+
+def full_text_search(text):
+    """
+    Helper function used for searching a taxonomy
+    """
+    normalized_text = re.sub(r"([^A-Za-z0-9])", r"\\\1", text) # Escape special characters
+    text_query_exact = "*" + normalized_text + '*'
+    text_query_fuzzy = normalized_text + "~"

```

Comment:
I think we should do a special version for "nodeSearchIds" to instead use the id normalization function.
So that if I search "Strawberry yogurt", it will search "stawberry-yogurt~" and "*stawberry-yogurt*"

Reply:
Where exactly would this query come? How much priority would we need to give for this?

Reply:
I just propose you to use a normalized version of text (using the same function as when you transform tag tot tag id) and build a text_id_query_exact and text_id_query_fuzzy and use as the parameter for the queryNodes on nodeSearchIds instead of text_query_exact and text_query_fuzzy

Reply:
It's because I suspect current version not to work on tags_id with multi word searches. (like "Straweberry yogurt")